egs2/dirha_wsj/asr1/run.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+mic=Beam_Circular_Array
+
+local_data_opts="--mic "${mic}" "
+
+stage=9
```

Comment:
make this stage as "1" when giving PR

Reply:
Okay, I'll fix this and other small bugs in CI today.